client/src/app/routes/dashboard/routes/tasks-route/tasks-route.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+
+@Component({
+  selector: 'app-tasks-route',
+  templateUrl: './tasks-route.component.html',
+  styleUrls: ['./tasks-route.component.scss'],
+})
+export class TasksRouteComponent implements OnInit {
+  baseUrlTasks: string | undefined;
+
+  constructor(private router: Router) {}
+
+  ngOnInit(): void {
+    const userId = localStorage.getItem('userId');
```

Comment:
It is very weird logic, please change auth mechanism at all. You should use guard and protect some routes by it

Reply:
What do you mean "weird logic"? Can you explain it a bit?